monitor/tests/unit/plugins/test_plugin.py

```diff
@@ -26,7 +26,7 @@
 
 class TestKubeJobs(unittest.TestCase):
 
-        
+
```

Comment:
unrelated.

Reply:
Modification made in commit 34fab04e5bd16c1e03a39164fa0c77717ad7cf3d